compiler/luci/pass/include/luci/Pass/ShapeInferencePass.h

```diff
@@ -19,21 +19,22 @@
 
 #include <loco.h>
 
-#include <logo/Pass.h>
+#include <luci/ModulePass.h>
 
 namespace luci
 {
 
 /**
  * @brief Pass to infer shape of nodes
  */
-class ShapeInferencePass : public logo::Pass
+class ShapeInferencePass : public luci::Pass
 {
 public:
   virtual const char *name(void) const { return "luci::ShapeInferencePass"; }
 
 public:
-  bool run(loco::Graph *graph);
+  bool run(luci::Module *m);
+  bool run(loco::Graph *g);
```

Comment:
do you have to change this line?

Reply:
Not have to, it was just for keeping consistency with `run(loco::Graph *g)` in `ShapeInferencePass.cpp`.
Shall I revert it?

Reply:
Yes please, for our existing ground rules; in another PR.